pkgs/servers/uwsgi/default.nix

```diff
@@ -61,6 +65,8 @@ stdenv.mkDerivation rec {
     sha256 = "0256v72b7zr6ds4srpaawk1px3bp0djdwm239w3wrxpw7dzk1gjn";
   };
 
+  patches = [ ./0001-no-ext-session-php_session.h-on-NixOS.patch ./0002-additional-php-ldflags.patch ];
```

Comment:
Can you break this line? It's to make the diff easier to read in the future.
Also, I would remove the number prefixes.